werf.yaml

```diff
@@ -269,13 +269,33 @@ ansible:
       echo -e "#!/bin/bash\nexec tini -- /deckhouse/deckhouse-controller/entrypoint.sh" > /deckhouse/deckhouse
       chmod +x /deckhouse/deckhouse
 
-  - name: "Run deckhouse from nobody"
+  # TODO: Remove after the next release
```

Comment:
Specify the release please. It will take some time to figure out what means `next`.
Better to set `Remove after the deckhouse release 1.55`